### PR TITLE
Remove helm snippets from On prem yamls

### PR DESCRIPTION
**Pull request by bobotimi**
[NEXUS-38171](https://issues.sonatype.org/browse/NEXUS-38171)

### CODE REVIEW

**Review by lisadurant — APPROVED** [review]

File: sample-on-prem-ha-yamls/onprem-ha-blobs-nfs-persistent-volume.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Copyright Sonatype, Inc., 2022, All Rights Reserved.
+#
+# Note: This YAML file is just an example and cannot be used as-is.
+# You must fill it out with the appropriate information for your deployment to be able to use it.
+
+kind: PersistentVolume
```
Comment (mpiggott) [inline sample-on-prem-ha-yamls/onprem-ha-blobs-nfs-persistent-volume.yaml:6]: The NFS mount is really optional, users may be using an appliance with an S3-style API.

File: sample-on-prem-ha-yamls/onprem-ha-docker-ingress.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+apiVersion: networking.k8s.io/v1
```
Comment (mpiggott) [inline sample-on-prem-ha-yamls/onprem-ha-docker-ingress.yaml:1]: Missing license header

File: sample-on-prem-ha-yamls/onprem-ha-nexus-data-local-persistent-volume.yaml [hunk 3]
```diff
@@ -0,0 +1,114 @@
+apiVersion: v1
```
Comment (mpiggott) [inline sample-on-prem-ha-yamls/onprem-ha-nexus-data-local-persistent-volume.yaml:1]: Missing license header

File: sample-on-prem-ha-yamls/onprem-ha-nexus-data-local-workdir-configmap.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```
Comment (mpiggott) [inline sample-on-prem-ha-yamls/onprem-ha-nexus-data-local-workdir-configmap.yaml:1]: Missing license header

File: sample-on-prem-ha-yamls/onprem-ha-nexus-data-local-workdir-daemonset.yaml [hunk 5]
```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
```
Comment (mpiggott) [inline sample-on-prem-ha-yamls/onprem-ha-nexus-data-local-workdir-daemonset.yaml:1]: Missing license header

File: sample-on-prem-ha-yamls/onprem-ha-services.yaml [hunk 6]
```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
```
Comment (mpiggott) [inline sample-on-prem-ha-yamls/onprem-ha-services.yaml:1]: Missing license header

File: sample-on-prem-ha-yamls/onprem-ha-storage-class.yaml [hunk 7]
```diff
@@ -0,0 +1,8 @@
+apiVersion: storage.k8s.io/v1
```
Comment (mpiggott) [inline sample-on-prem-ha-yamls/onprem-ha-storage-class.yaml:1]: Missing license header

File: sample-on-prem-ha-yamls/onprem-ha-nexus-data-local-persistent-volume.yaml (hunk 3 above)
Comment (bobotimi) [reply sample-on-prem-ha-yamls/onprem-ha-nexus-data-local-persistent-volume.yaml:6]: Done

File: sample-on-prem-ha-yamls/onprem-ha-namespaces.yaml [hunk 8]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```
[the remark this answers is not in the document]
Comment (bobotimi) [reply sample-on-prem-ha-yamls/onprem-ha-namespaces.yaml:6]: Done

**Review by oleksiirudyk — APPROVED** [review]

**Pull request merged**